prefs.js

```diff
@@ -380,6 +374,65 @@ class SettingsWidget {
             this._settings.set_value('winprops', new GLib.Variant('as', rows));
         });
 
+        // Advanced
+        booleanSetState('gesture-enabled');
+
+        const gestureFingers = key => {
+            const builder = this.builder.get_object(key);
+            const setting = this._settings.get_int(key);
+            switch (setting) {
+            case 3:
+                builder.set_active_id('three-fingers');
+                break;
+            case 4:
+                builder.set_active_id('four-fingers');
+                break;
+            default:
+                builder.set_active_id('fingers-disabled');
+            }
```

Comment:
I think this is slightly better using a lookup table. E.g.

```suggestion
            const lookupTable = {
                3: 'three-finters',
                4: 'four-fingers'
            };
            builder.set_active_id(lookupTable[setting] ?? 'finger-disabled');
```

You can also generate the inverse lookup table with e.g.

```
Object.fromEntries(Object.entries(obj).map(a => a.reverse()))
```

or write it by hand (which might be slightly less magic^^).

Same goes for the other switch statements.

Reply:
Done - yeah, is much more succinct.  Thanks!